src/client/eppo-client.ts

```diff
@@ -543,14 +562,24 @@ export default class EppoClient {
 
       if (banditKey) {
         evaluationDetails.banditKey = banditKey;
-        action = this.evaluateBanditAction(
+        const banditEvent = this.evaluateBanditAction(
           flagKey,
           subjectKey,
           subjectAttributes,
           actions,
           banditKey,
           evaluationDetails,
+          selectBestAction,
         );
+        action = banditEvent?.action ?? null;
+        if (banditEvent !== null) {
+          try {
+            this.logBanditAction(banditEvent);
```

Comment:
we should not log if `selectBestAction` is true

Reply:
Logging removed from getBestBanditAction codepath entirely